### PR TITLE
Introduced exoAPI : an API interface to get data from RESTful API seemlessly

### PR DESCRIPTION
**Pull request by Shankhanil**

### CODE REVIEW

**Comment by Shankhanil** [comment]
@chhandasaha  Pls review the commit !

File: src/Quotes.py [hunk 1]
```diff
@@ -28,6 +29,40 @@ def formatQuote(self, quoteTuple):
...
+        pass
+        
+class Quotes2:
+    '''
+    A Quotes v2.0 : using the exoAPI interface
+    '''
+    def __init__(self):
+        self.api = exoREST()
+    
+    def generateQuote(self, author = 'random'):
```
Comment (chhandasaha) [inline src/Quotes.py:43]: What is the purpose of  author = 'random'

File: src/Quotes.py [hunk 2]
```diff
@@ -28,6 +29,40 @@ def formatQuote(self, quoteTuple):
...
+                url = 'https://quote-garden.herokuapp.com/api/v2/quotes/random')
+        else:
+            self.api.addAPI_URL(url_name = author, \
+                url = 'https://quote-garden.herokuapp.com/api/v2/authors/{}'.format(author))
+        
+        return self.api.getDataAsJSON(url_name = author, \
+                method = 'get', outputformat = 'json')
+        
+    def getQuote(self, quoteJSON):
+        params = ['quote/quoteText', 'quote/quoteAuthor']
```
Comment (chhandasaha) [inline src/Quotes.py:55]: What is the use of 'params' ?

File: src/Quotes.py (hunk 1 above)
Comment (Shankhanil) [reply src/Quotes.py:43]: to get a quote from any random author. 
<i> Put in documentation </i>

File: src/Quotes.py (hunk 2 above)
Comment (Shankhanil) [reply src/Quotes.py:55]: dict Parameters : quote/quoteText and quote/quoteAuthor

**Pull request merged**